hack/install-golangci-lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Why `hack` dir? :)

Reply:
This is the way.

Reply:
Changed it to `.openshiftci`. This one is required to "store" some CI scripts. We may end up having more in the future, so... better to have a separate space to store it.